app/components/NotificationsArea.vue.ts

```diff
@@ -0,0 +1,86 @@
+import Vue from 'vue';
+import { Component } from 'vue-property-decorator';
+import { Inject } from 'util/injector';
+import {
+  ENotificationType,
+  NotificationsService,
+  INotification
+} from 'services/notifications';
+
+const LIFE_TIME = 15000;
+
+interface IUiNotification extends INotification {
+  outdated?: boolean;
+}
+
+@Component({})
+export default class NotificationsArea extends Vue {
```

Comment:
Did you ever set up Prettier on your editor?  I think we should be running it on new files.  It will collapse multiple line endings in a row, which I know is a style you like.  However, I think in general as we have more and more people touching the JS it's just easier if everybody is running the same formatter.

Reply:
Oh, I thought Prettier does some changes before commit.. Actually the separating methods with 2 lines makes code more readable and helps to resolve merge conflicts since we use one-line spacing to separate logic blocks in one method. Maybe we can setup Prettier to do that? I know that some code combs have this option

Reply:
I have not enabled any pre-commit hooks for prettier yet, since that seemed a little aggressive.  I will probably end up enabling that soon.

Unfortunately it looks like prettier does not allow that as an option.  Personally I'm not convinced adding an additional newline necessarily makes it more readable.  Between indentation and TypeDoc comments, it's usually pretty easy at a glance to tell where top level functions begin and end.  Personally I'm not convinced that this particular issue is a big enough deal to stop using prettier.